enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java

```diff
@@ -239,13 +330,12 @@ private void applyQueuedTransactions() throws IOException
                         // threads waiting for transaction obligations to be fulfilled and since they are looking
                         // at last closed transaction id they might get notified to continue before all data
                         // has actually been flushed properly.
-                        transactionQueue.accept( batchCloser );
+                        transactionQueue.accept( batchCloser ); // MARK TXs AS CLOSED
```

Comment:
Be aware that this part is _completely_ different in 3.0. So there will likely be a lot of changes in to this and other classes in the merge commit.